packages/big-design/src/components/Dropdown/Dropdown.tsx

```diff
@@ -146,6 +147,7 @@ export const Dropdown = memo(
       (group: DropdownItemGroup) => {
         return (
           <>
+            {group.separated && <HR margin="none" />}
```

Comment:
You'll need to throw this into a `aria-hidden` `li` to make it semantic & accessible. Follow a similar pattern to `ListGroupHeader`.